Automattic/woocommerce-payments
# Add urls for the link to the documentation page

Fixes #1780 

#### Changes proposed in this Pull Request
Add the URL to the docs link and remove inline TODO items.








#### Testing instructions

- Visit the multi-currency settings page and verify that the "Learn More" link at the top takes you to the documentation page.





*

-------------------
